autogluon/task/tabular_prediction/hyperparameter_configs.py

```diff
@@ -0,0 +1,49 @@
+
+import copy
+
+
+# Dictionary of preset hyperparameter configurations.
+hyperparameter_config_dict = dict(
+    # Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
+    default={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {},
+        'XT': {},
+        'KNN': {},
+        'custom': ['GBM'],
+    },
+    # Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
+    light={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {'max_depth': 15},
+        'XT': {'max_depth': 15},
+        'custom': ['GBM'],
+    },
+    # Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
+    very_light={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+    },
+    # Results in extremely small models. Only use this when prototyping, as the model quality will be severely reduced.
+    toy={
+        'NN': {'num_epochs': 10},
```

Comment:
Note NN is not actually smaller here, reducing num_epochs just makes it train faster.  If we want extremely small-size NN, can do something like:

'NN': {'max_layer_width': 128, 'max_embedding_dim':8, 'embedding_size_factor': 0.25}

Reply:
renamed to 'quick to train'